ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -23,21 +24,31 @@ class DoctorCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--report', '-r', action='store_true',
-            help='Print out doctor report.'
+            help='Print out all report info.'
+        )
+        parser.add_argument(
+            '--report_failed', '-rf', action='store_true',
+            help='Print out report info on failed checks.'
         )
 
     def main(self, *, parser, args):
+        # print report only if no failed checks requested
         if args.report:
-            generate_report()
+            report = generate_report()
+            format_print(report.keys(), report)
```

Comment:
`(report.keys(), report)` is passing the same information twice. Can this be just `(report)`?